docker-compose.yml

```diff
@@ -162,7 +162,7 @@ services:
   db:
     image: mariadb:10.3.10-bionic
     restart: unless-stopped
-    container_name: kopano_db
+    container_name: ${COMPOSE_PROJECT_NAME}-kopano_db
```

Comment:
docker networks use ${COMPOSE_PROJECT_NAME}_network-name as a format. if this is done for consistency, then this should imho also use an underscore here as well.

Reply:
On a related note, make it rather ${COMPOSE_PROJECT_NAME}_db than the current one. else we would currently end up with kopano-kopano_db as the container name in this example.

Reply:
Also changed for ssl, server, webapp, zpush, grapi, kapi, kdav, dagent, spooler, gateway, ical, monitor, search, konnect, playground, kwmserver and meet.